scripts/integration_test.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
 # get Go package list from caller
-PACKAGES=$1
+PACKAGES=$@
```

Comment:
Changed here since `$1` only matches to the first argument, not all of them (for handling multiple sub-packages)